`mvn site` fails for 3.0

## Environment Details
* Helidon Version: 3.0
* Helidon SE or Helidon MP
* JDK version:
* OS:
* Docker version (if applicable):

----------

## Problem Description
`mvn site` fails for me (and @barchetta) with
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.8.2:site (default-site) on project helidon-project: Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.8.2:site failed: An API incompatibility was encountered while executing org.apache.maven.plugins:maven-site-plugin:3.8.2:site: java.lang.IllegalAccessError: class net.sourceforge.plantuml.png.PngIOMetadata (in unnamed module @0x247be2e5) cannot access class com.sun.imageio.plugins.png.PNGMetadata (in module java.desktop) because module java.desktop does not export com.sun.imageio.plugins.png to unnamed module @0x247be2e5
```
and then a long list of URLs.

## Steps to reproduce
1. Switch to `master`.
2. `mvn site`

```
Caused by: java.lang.IllegalAccessError: class net.sourceforge.plantuml.png.PngIOMetadata (in unnamed module @0x3a90dbf) cannot access class com.sun.imageio.plugins.png.PNGMetadata (in module java.desktop) because module java.desktop does not export com.sun.imageio.plugins.png to unnamed module @0x3a90dbf
    at net.sourceforge.plantuml.png.PngIOMetadata.writeWithMetadata (net/sourceforge/plantuml/png/PngIOMetadata.java:50)
    at net.sourceforge.plantuml.png.PngIO.write (net/sourceforge/plantuml/png/PngIO.java:76)
    at net.sourceforge.plantuml.png.PngIO.write (net/sourceforge/plantuml/png/PngIO.java:70)
    at net.sourceforge.plantuml.ugraphic.g2d.UGraphicG2d.writeImageTOBEMOVED (net/sourceforge/plantuml/ugraphic/g2d/UGraphicG2d.java:206)
    at net.sourceforge.plantuml.ugraphic.ImageBuilder.writeImageInternal (net/sourceforge/plantuml/ugraphic/ImageBuilder.java:239)
    at net.sourceforge.plantuml.ugraphic.ImageBuilder.writeImageTOBEMOVED (net/sourceforge/plantuml/ugraphic/ImageBuilder.java:161)
    at net.sourceforge.plantuml.svek.CucaDiagramFileMakerSvek.createFileInternal (net/sourceforge/plantuml/svek/CucaDiagramFileMakerSvek.java:114)
    at net.sourceforge.plantuml.svek.CucaDiagramFileMakerSvek.createFile (net/sourceforge/plantuml/svek/CucaDiagramFileMakerSvek.java:60)
    at net.sourceforge.plantuml.cucadiagram.CucaDiagram.exportDiagramInternal (net/sourceforge/plantuml/cucadiagram/CucaDiagram.java:322)
    at net.sourceforge.plantuml.classdiagram.ClassDiagram.exportDiagramInternal (net/sourceforge/plantuml/classdiagram/ClassDiagram.java:197)
    at net.sourceforge.plantuml.UmlDiagram.exportDiagramNow (net/sourceforge/plantuml/UmlDiagram.java:220)
    at net.sourceforge.plantuml.AbstractPSystem.exportDiagram (net/sourceforge/plantuml/AbstractPSystem.java:117)
    at net.sourceforge.plantuml.SourceStringReader.generateImage (net/sourceforge/plantuml/SourceStringReader.java:115)
    at net.sourceforge.plantuml.SourceStringReader.generateImage (net/sourceforge/plantuml/SourceStringReader.java:98)
    at org.asciidoctor.diagram.PlantUML.generate (org/asciidoctor/diagram/PlantUML.java:99)
    at org.asciidoctor.diagram.CommandProcessor.processRequest (org/asciidoctor/diagram/CommandProcessor.java:32)
    at org.asciidoctor.diagram.CommandProcessor.processRequest (org/asciidoctor/diagram/CommandProcessor.java:11)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (jdk/internal/reflect/NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (jdk/internal/reflect/DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (java/lang/reflect/Method.java:568)
    at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling (org/jruby/javasupport/JavaMethod.java:455)
    at org.jruby.javasupport.JavaMethod.invokeDirect (org/jruby/javasupport/JavaMethod.java:316)
    at RUBY.send_request (uri:classloader:/gems/asciidoctor-diagram-1.5.4.1/lib/asciidoctor-diagram/util/java_jruby.rb:24)
    at RUBY.plantuml (uri:classloader:/gems/asciidoctor-diagram-1.5.4.1/lib/asciidoctor-diagram/plantuml/extension.rb:37)
    at RUBY.included (uri:classloader:/gems/asciidoctor-diagram-1.5.4.1/lib/asciidoctor-diagram/plantuml/extension.rb:82)
    at org.jruby.RubyBasicObject.yieldUnder (org/jruby/RubyBasicObject.java:1762)
    at org.jruby.RubyBasicObject.instance_exec (org/jruby/RubyBasicObject.java:2666)
    at org.jruby.RubyBasicObject$INVOKER$i$0$3$instance_exec.call (org/jruby/RubyBasicObject$INVOKER$i$0$3$instance_exec.gen)
    at RUBY.create_image_block (uri:classloader:/gems/asciidoctor-diagram-1.5.4.1/lib/asciidoctor-diagram/extensions.rb:172)
    at RUBY.process (uri:classloader:/gems/asciidoctor-diagram-1.5.4.1/lib/asciidoctor-diagram/extensions.rb:115)
    at org.jruby.RubyMethod.call (org/jruby/RubyMethod.java:125)
    at org.jruby.RubyMethod$INVOKER$i$call.call (org/jruby/RubyMethod$INVOKER$i$call.gen)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.invokeOther64:\=\{\} (uri_3a_classloader_3a_/gems/asciidoctor_minus_1_dot_5_dot_8/lib/asciidoctor/uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:1038)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.build_block (uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:1038)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.invokeOther145:build_block (uri_3a_classloader_3a_/gems/asciidoctor_minus_1_dot_5_dot_8/lib/asciidoctor/uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:866)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.next_block (uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:866)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.invokeOther102:next_block (uri_3a_classloader_3a_/gems/asciidoctor_minus_1_dot_5_dot_8/lib/asciidoctor/uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:369)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.next_section (uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:369)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.invokeOther84:next_section (uri_3a_classloader_3a_/gems/asciidoctor_minus_1_dot_5_dot_8/lib/asciidoctor/uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:354)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.next_section (uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:354)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.invokeOther5:next_section (uri_3a_classloader_3a_/gems/asciidoctor_minus_1_dot_5_dot_8/lib/asciidoctor/uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:98)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.parser.parse (uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/parser.rb:98)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.document.invokeOther36:parse (uri_3a_classloader_3a_/gems/asciidoctor_minus_1_dot_5_dot_8/lib/asciidoctor/uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/document.rb:570)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.document.parse (uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/document.rb:570)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.document.invokeOther8:parse (uri_3a_classloader_3a_/gems/asciidoctor_minus_1_dot_5_dot_8/lib/asciidoctor/uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/document.rb:1171)
    at uri_3a_classloader_3a_.gems.asciidoctor_minus_1_dot_5_dot_8.lib.asciidoctor.document.convert (uri:classloader:/gems/asciidoctor-1.5.8/lib/asciidoctor/document.rb:1171)
    at org.jruby.RubyClass.finvoke (org/jruby/RubyClass.java:900)
    at org.jruby.RubyBasicObject.callMethod (org/jruby/RubyBasicObject.java:370)
    at org.asciidoctor.internal.RubyObjectWrapper.getRubyProperty (org/asciidoctor/internal/RubyObjectWrapper.java:131)
    at org.asciidoctor.internal.RubyObjectWrapper.getString (org/asciidoctor/internal/RubyObjectWrapper.java:37)
    at org.asciidoctor.ast.impl.StructuralNodeImpl.convert (org/asciidoctor/ast/impl/StructuralNodeImpl.java:92)
    at io.helidon.build.sitegen.asciidoctor.AsciidocEngine.render (io/helidon/build/sitegen/asciidoctor/AsciidocEngine.java:248)
    at io.helidon.build.sitegen.asciidoctor.AsciidocPageRenderer.process (io/helidon/build/sitegen/asciidoctor/AsciidocPageRenderer.java:62)
    at io.helidon.build.sitegen.RenderingContext.processPages (io/helidon/build/sitegen/RenderingContext.java:143)
    at io.helidon.build.sitegen.VuetifyBackend.generate (io/helidon/build/sitegen/VuetifyBackend.java:135)
    at io.helidon.build.sitegen.Site.generate (io/helidon/build/sitegen/Site.java:128)
    at io.helidon.build.sitegen.maven.DoxiaSiteRenderer.render (io/helidon/build/sitegen/maven/DoxiaSiteRenderer.java:111)
    at org.apache.maven.plugins.site.render.SiteMojo.renderLocale (org/apache/maven/plugins/site/render/SiteMojo.java:194)
```

--- a/pom.xml
+++ b/pom.xml
@@ -55,7 +55,7 @@
         Changing these version requires approval for a new third party dependency!
         -->
         <version.lib.arquillian>1.7.0.Alpha10</version.lib.arquillian>
-        <version.lib.asciidoctor.diagram>1.5.4.1</version.lib.asciidoctor.diagram>
+        <version.lib.asciidoctor.diagram>1.5.18</version.lib.asciidoctor.diagram>
         <version.lib.asm>6.0</version.lib.asm>
         <version.lib.checkstyle>9.1</version.lib.checkstyle>
         <version.lib.groovy-all>2.4.14</version.lib.groovy-all>

